gen_mod_headers

```diff
@@ -219,24 +216,17 @@ if [[ -n "$prefix" ]]; then
 	sed -i 's|scripts/basic/fixdep|xscripts/basic/fixdep|' scripts/Makefile.build
 	sed -i 's|$(obj)/mk_elfconfig|x$(obj)/mk_elfconfig|' scripts/mod/Makefile
 	sed -i 's|$(obj)/conf|x$(obj)/conf|' scripts/kconfig/Makefile
-	if [[ -e tools/build/Makefile ]]; then
-		sed -i 's|$(QUIET_LINK)$(HOSTCC) $(LDFLAGS) -o $@ $<|$(QUIET_LINK)$(HOSTCC) $(LDFLAGS) -o fixdep_native $<|' tools/build/Makefile
-		# trying to apply the previous sed in context of 4.18 kernel
-		sed -i 's|$(QUIET_LINK)$(HOSTCC) $(HOSTLDFLAGS) -o $@ $<|$(QUIET_LINK)$(HOSTCC) $(HOSTLDFLAGS) -o fixdep_native $<|' tools/build/Makefile
-	fi
-
-	# copy objtools fixdep
-	if [[ -e xtools/objtool/fixdep ]]; then
-		cp xtools/objtool/fixdep tools/objtool/
-	fi
 
 	echo Building script binaries for target arch...
 	make HOSTCC="$hostcc" $build_opts scripts
 
-	# copy the native fixdep back
-	if [[ -e tools/build/fixdep_native ]]; then
-		mv tools/build/fixdep_native tools/objtool/fixdep
-	fi
+        if [[ -e tools/build/Makefile ]]; then
+            echo Building objtool fixdep for target
+            make -C tools/build HOSTCC="$hostcc" $build_opts
```

Comment:
to build `objtool`, it used to need `fixdep`. And that was what would break on the builders. As it would build using the local machines `fixdep`. (I think that was an issue from memory)

Reply:
Have you tried running a manual jenkins job with this?
wip commit that uses this branch in gen_mod_headers recipe. and then a wip PR so that all devices are built for?

Reply:
Yes and all device repos built fine